docs/tutorials/mip_convert.md

```diff
@@ -186,22 +186,25 @@ MIP Convert determines:
+|---------------------------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------|
+| `ancil_files`                         |          | A space separated list of the full paths to any required ancillary files.                                                                                                              |        |
+| `atmos_timestep`                      |          | The atmospheric model timestep in integer seconds.                                                                                                                                     | [1]    |
+| `base_date`                           | Yes      | The date in the form `YYYY-MM-DDThh:mm:ss`.                                                                                                                                            | [2]    |
+| `deflate_level`                       |          | The deflation level when writing the output netCDF file from 0 (no compression) to 9 (maximum compression).                                                                            |        |
+| `force_coordinate_rotation`           |          | If set to `True`, output data will be forced to include rotated coordinates and true lat-lon coordinates.                                                                              |        |
+| `hybrid_heights_file`                 |          | A space separated list of the full path to the files containing the information about the hybrid heights.                                                                              | [3]    |
+| `mask_slice`                          | Yes      | Optional slicing expression for masking data in the form of `n:m,i:j`, or `no_mask`                                                                                                    | [4][8] |
+| `model_output_dir`                    | Yes      | The full path to the root directory containing the model output files.                                                                                                                 | [5]    |
+| `mip_convert_plugin`                  | Yes      | The id of the MIP convert plugin that should be used.                                                                                                                                  | [5]    |
+| `mip_convert_external_plugin`         |          | The module path to external MIP convert plugin, e.g. `cdds_arise.arise_mip_convert_plugin`.                                                                                            |        |
+| `mip_convert_external_plugin_location`|          | The full path to the external plugin implementation, e.g. `/home/users/cdds/arise`.                                                                                                    |        |
```

Comment:
We shouldn't use a "real" location here, perhaps `$CDDS_ETC/mapping_plugins/arise` instead?